modules/account-lib/src/coin/hbar/transaction.ts

```diff
@@ -72,7 +94,7 @@ export class Transaction extends BaseTransaction {
       memo: this._txBody.memo,
     };
 
-    if (this._txBody.data === 'cryptoTransfer') {
+    if (this._txBody.data === HederaTransactionTypes.Transfer) {
```

Comment:
just wanna ensure that the PR with the test also include the change for account initialization right?